libs/graph-lib/src/services/graph-state-manager.ts

```diff
@@ -115,6 +115,17 @@ export class GraphStateManager implements OnApplicationBootstrap {
     return privacyType;
   }
 
+  public getConnectionTypeForSchema(schemaId: number): ConnectionType {
+    let connectionType = ConnectionType.Friendship;
+    if (
+      this.schemaIds[ConnectionType.Follow][PrivacyType.Private] === schemaId ||
+      this.schemaIds[ConnectionType.Follow][PrivacyType.Public] === schemaId
+    ) {
+      connectionType = ConnectionType.Follow;
+    }
+    return connectionType;
```

Comment:
Just some code golf if you want a more generic way of doing this; not really necessary since there's only 2 entries at each level here, but if you're interested:
```suggestion
return Object.entries(this.schemaId).filter(([connectionType, v]) => Object.values(v).includes(schemaId)).map(([connectionType, v]) => connectionType)[0] || ConnectionType.Friendship;
```

Frankly, yours is more readable, but I still thought this was cool.

Reply:
Thanks for the suggestion. I think it would make it harder to read and understand.